proxy/http/HttpSM.cc

```diff
@@ -6597,6 +6597,11 @@ HttpSM::setup_error_transfer()
 void
 HttpSM::setup_internal_transfer(HttpSMHandler handler_arg)
 {
+  if (ua_entry == nullptr || ua_entry->vc == nullptr) {
```

Comment:
Don't we need to call `HTTP_SM_SET_DEFAULT_HANDLER(handler_arg)` even if a client is already gone?
https://github.com/apache/trafficserver/blob/15bea4dd946c8cb6fc2000a4b31cf4f2f261b29d/proxy/http/HttpSM.cc#L1769-L1771

Reply:
Ah, in that case, it looks like `HttpSM::setup_server_transfer_to_cache_only();` should be called eventually.
OK, I'll go with another approach.

Reply:
Yeah, that's what I thought.